components/contexts/Web3Context.js

```diff
@@ -4,15 +4,17 @@ import { ethers } from 'ethers'
 import Web3Modal from 'web3modal'
 
 import LvlV1ABI from '../../abi/contracts/LvlV1.sol/LvlV1.json'
-import { InfuraId, LvlV1Address, Network } from '../../util/constants'
+import { LvlV1Address, Network, RPCPath } from '../../util/constants'
 
 const Web3Context = createContext()
 
 const providerOptions = {
   walletconnect: {
     package: WalletConnectProvider,
     options: {
-      infuraId: InfuraId,
+      rpc: {
+        [Network.id]: `https://${RPCPath}`,
+      },
```

Comment:
Switched Infura to Alchemy because it's more reliable and gives us better tooling.